### ISSUE TITLE
[bug] HomePage slugs wrong at new site creation

### ISSUE THREAD

**Issue by denbatte** [Kunstmaan/KunstmaanBundlesCMS#589]
When following the documentation with the --demosite option, you get a error of a route not found.
You are able to access the admin area though.

I noticed a strange thing in the menu - slug field.
It says /{locale}/n-a.
If you remove the n-a value, everything works fine.

![screen shot 2015-07-27 at 22 19 56](https://cloud.githubusercontent.com/assets/505196/8916744/e443c71a-34ad-11e5-8d3b-190769496ac9.png)

@krispypen: You can assign me to this issue.

Edit: Tracked down to https://github.com/Kunstmaan/KunstmaanUtilitiesBundle/blob/master/Helper/Slugifier.php#L18

**Comment by roderik** [comment]
Also with 3.2.2?

**Comment by denbatte** [comment]
Yes

**Comment by JoakimLofgren** [comment]
I have issues with this as well.

I noticed in my fixtures that when I change `setSlug('')` to `setSlug(null)`:
```php
$translations = array();
$translations[] = array(
    'language' => 'en',
    'callback' => function ($page, $translation, $seo) {
        $translation->setTitle('Home page');
        $translation->setSlug(null);
        $translation->setUrl($translation->getFullSlug());
        $translation->setWeight(-50);
    }
);
```

Then the slug for my root page is /en (or empty), but with '' it becomes /en/n-a.

**Comment by denbatte** [comment]
I think this might be fixed with @krispypen pull request: #603. Need to test to verify though.
